Twitch/Twitch.Base/Models/Clients/Chat/ChatUserNoticePacketModel.cs

```diff
@@ -155,6 +155,8 @@ public class ChatUserNoticePacketModel : ChatUserPacketModelBase
         /// </summary>
         public string RitualName { get; set; }
 
+        public int SubBenefitEndMonth { get; set; }
```

Comment:
Include a comment explaining what the property entails & move closer to the other sub-related properties.